paper-char-counter.html

```diff
@@ -94,13 +94,27 @@
     /* True if the number of characters in the input exceeds _maxChars */
     _isCounterInvalid: false,
 
-    ready: function() {
-      if (!this.target)
-        return;
-      var targetElement = document.getElementById(this.target);
-      this._maxChars = targetElement.maxLength;
-      targetElement.addEventListener('input', this.inputAction.bind(this));
-		},
+    _targetElement: null,
+
+    ready: function () {
+        if (!this.target)
+            return;
+
+        this._initTarget(document.getElementById(this.target));
+    },
+
+    _initTarget: function (targetElement) {
+        this._targetElement = targetElement;
+
+        this._maxChars = this._targetElement.maxLength;
+        this._targetElement.addEventListener('input', this.inputAction.bind(this));
+    },
+
+    targetChanged: function (o, n) {
+        if (n && !this._targetElement) {
+            this._initTarget(typeof n == 'string' ? document.getElementById(n) : n);
```

Comment:
You are right, but I throw in the towel regarding to small amount of interactions between project leads and external PR... sorry. By the way I switched back to AngularJS v1, too much fork were required to make polymer used in production.